### PR TITLE
JavascriptOperators::GetTypeId overload with parameter type RecyclableObject*

### PR DESCRIPTION
**Pull request by rajatd**

### CODE REVIEW

**Comment by obastemur** [comment]
LGTM

File: lib/Runtime/Language/JavascriptOperators.inl [hunk 1]
```diff
@@ -31,6 +31,18 @@ namespace Js
         }
     }
 
+    __forceinline TypeId JavascriptOperators::GetTypeId(Js::RecyclableObject* aValue)
+    {
+        AssertMsg(aValue != nullptr, "GetTypeId aValue is null");
+
+        auto typeId = RecyclableObject::UnsafeFromVar(aValue)->GetTypeId();
```
Comment (curtisman) [inline lib/Runtime/Language/JavascriptOperators.inl]: Is the UnsafeFromVar needed here?  Or is it there just to get the Assert in UnsafeFromVar?

**Comment by MikeHolman** [comment]
I actually already made this exact change as part of my JSON changes (since this is called in a hot path).
https://github.com/Microsoft/ChakraCore/pull/4077/files#diff-997a2a4e9717a6d1f7b68d9676805309

**Comment by rajatd** [comment]
Already part of #4077

**Pull request closed without merging**